docs/source/en/tasks/sequence_classification.mdx

```diff
@@ -197,18 +251,163 @@ Configure the model for training with [`compile`](https://keras.io/api/models/mo
+
+Set up your `compute_metrics` function with [`~transformers.KerasMetricCallback`]:
+
+```py
+>>> from transformers.keras_callbacks import KerasMetricCallback
+
+>>> metric_callback = KerasMetricCallback(metric_fn=compute_metrics, eval_dataset=tf_validation_set)
+```
+
+Set up the [`transformers.PushToHubCallback`]:
 
 ```py
->>> model.fit(x=tf_train_set, validation_data=tf_validation_set, epochs=3)
+>>> from transformers.keras_callbacks import PushToHubCallback
+
+>>> push_to_hub_callback = PushToHubCallback(
+...     output_dir="my_awesome_model",
+...     tokenizer=tokenizer,
+... )
+```
+
+Then bundle your callbacks together and pass them to [`fit`] in the next step:
+
+```py
+>>> callbacks = [metric_callback, push_to_hub_callback]
+```
+
+Finally, you're ready to start training your model! Call [`fit`] with your training and validation datasets, the number of epochs, and your callbacks:
+
+Call [`fit`](https://keras.io/api/models/model_training_apis/#fit-method) to finetune the model:
+
+```py
+>>> model.fit(x=tf_train_set, validation_data=tf_validation_set, epochs=3, callbacks=callbacks)
 ```
 </tf>
 </frameworkcontent>
 
 <Tip>
 
-For a more in-depth example of how to fine-tune a model for text classification, take a look at the corresponding
+For a more in-depth example of how to finetune a model for text classification, take a look at the corresponding
 [PyTorch notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/text_classification.ipynb)
 or [TensorFlow notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/text_classification-tf.ipynb).
 
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Inference
+
+Great, now that you've finetuned a model, you can use it for inference!
+
+Grab some text you'd like to run inference on:
+
+```py
+>>> text = "This was a masterpiece. Not completely faithful to the books, but enthralling from beginning to end. Might be my favorite of the three."
+```
+
+The simplest way to try out your finetuned model for inference is to use it in a [`pipeline`]. Map the expected id to their labels with `id2label` and `label2id`:
+
+```py
+>>> id2label = {0: "NEGATIVE", 1: "POSITIVE"}
+>>> label2id = {"NEGATIVE": 0, "POSITIVE": 1}
+```
+
+<frameworkcontent>
+<pt>
+```py
+>>> from transformers import pipeline, AutoTokenizer, AutoModelForSequenceClassification
+
+>>> tokenizer = AutoTokenizer.from_pretrained("stevhliu/my_awesome_model")
+>>> model = AutoModelForSequenceClassification.from_pretrained(
+...     "stevhliu/my_awesome_model", id2label=id2label, label2id=label2id
+... )
+```
+
+Instantiate the `pipeline` for sentiment analysis with your model and tokenizer, and pass your text to it:
+
+```py
+>>> classifier = pipeline("sentiment-analysis", model=model, tokenizer=tokenizer)
```

Comment:
```suggestion
>>> classifier = pipeline("sentiment-analysis", model="stevhliu/my_awesome_model")
```
If labels are handled previously, you'll be able to do just this, which is way shorted (and framewok-agnostic).